Cargo.toml

```diff
@@ -79,7 +79,7 @@ sqlx = { version = "0.8.2", features = [
   "chrono",
   "postgres",
 ] }
-strum = { version = "0.26", features = ["derive"] }
+strum = { version = "0.27", features = ["derive"] }
```

Comment:
question: I assume this is fine and doesn't introduce anything breaking?

Reply:
Yep :) (WCGW?)